Makefile

```diff
@@ -39,77 +47,73 @@ functional: ginkgo
 ifeq (${RUNTIME},)
 	$(error RUNTIME is not set)
 else
-	./ginkgo -v functional/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
-	bash sanity/check_sanity.sh
+	$(QUIET_TEST) ./ginkgo -v functional/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
+	$(QUIET_TEST) bash sanity/check_sanity.sh
 endif
 
 docker: ginkgo
 ifeq ($(RUNTIME),)
 	$(error RUNTIME is not set)
 else
-	./ginkgo -v -focus "${FOCUS}" -skip "${SKIP}" ./integration/docker/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
-	bash sanity/check_sanity.sh
+	$(QUIET_TEST) ./ginkgo --succinct --trace -focus "${FOCUS}" -skip "${SKIP}" ./integration/docker/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
+	$(QUIET_TEST) bash sanity/check_sanity.sh
 endif
 
 crio:
-	bash .ci/install_bats.sh
-	RUNTIME=${RUNTIME} ./integration/cri-o/cri-o.sh
+	$(QUIET_TEST) bash .ci/install_bats.sh
+	$(QUIET_TEST) bash -c "RUNTIME=${RUNTIME} && ./integration/cri-o/cri-o.sh"
 
 docker-compose:
-	bash .ci/install_bats.sh
-	cd integration/docker-compose && \
-	bats docker-compose.bats
+	$(QUIET_TEST) bash .ci/install_bats.sh
+	$(QUIET_TEST) bash -c "cd integration/docker-compose && bats docker-compose.bats"
```

Comment:
what about: 
```
bash -c "bats integration/docker-compose/docker-compose.bats"
```

Reply:
Oh, this is really frustrating.... can I scream yet? :-/
ouch ouch ouch.... so, it seems the `&&`'s in the Makefile, when they hit the eval, are separated out as a separate (non-eval'd) command. and hence, it fails. You can see it if you add a `set -x` to the kata_chronic.sh script. Oh, and the Makefile still works fine with `CI=false`. grrrr.

All uses of `&&` in the Makefile were for `bats`, and luckily it looks like converting them to a direct `bats the/full/path` works, so I've done that.
Of course, then I was concerned that all use of `||` and `;` in the Makefile would have a similar problem. I think I've managed to convert the use of `;`, and the use of `||` I think will work (although maybe not be so quiet), as the return value from the initial `QUIET` call will still be valid.
sigh. OK, let's spin the wheel again....